index.html

```diff
@@ -264,7 +264,52 @@ <h4 class="dark"> <span class="figure-3"></span> Skills </h4>
     </div>
 
     <div class="card">
+      <div class="card-contact-me">
+        <section>
```

Comment:
- [ ] **OPTIONAL** Please keep in mind that the container of the section should be a `<section>` tag, and not a `<div>` tag first!
According to the [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp), it'd be way too better for the website reading and optimizing.

Reply:
necessary changes were made 👍